src/obr/cli.py

```diff
@@ -136,25 +137,27 @@ def submit(ctx: click.Context, **kwargs):
             selected_jobs: list[Job] = [
                 j for j in project if bundle_value in list(j.sp().values())
             ]
-            print(f"[OBR] submit bundle {bundle_value} of {len(selected_jobs)} jobs")
-            print(
-                "[OBR] submission response",
+            logging.info(
+                f"[OBR] submit bundle {bundle_value} of {len(selected_jobs)} jobs"
```

Comment:
```suggestion
                f"Submit bundle {bundle_value} of {len(selected_jobs)} jobs"
```
Remove the [OBR] and rather set it via the format string, https://docs.python.org/3/library/logging.html#logging.basicConfig

Reply:
Done. Logs will be printed to console in the following format: 
```
λ git case_windsor_body_plots* → obr run -o fetchCas

[obr.cli:45]	  INFO: Specified operation fetchCas is not a valid operation.
[obr.cli:26]	  INFO: Available operations are:
	apply
	archive
	blockMesh
	checkMesh
	controlDict
	decomposePar
	fetchCase
	fvSolution
	refineMesh
	runParallelSolver
	runSerialSolver
	setKeyValuePair
	shell
```